src/components/CircularButton.tsx

```diff
@@ -5,17 +5,26 @@ type Direction = "left" | "top" | "right";
 export interface ICircularButtonProps {
   direction?: Direction;
   size?: number;
+  className?: string;
+  handlerOnClick?: (newVal?) => void;
 }
-
-const CircularButton: FunctionComponent<ICircularButtonProps> = (props) => {
+const CircularButton: FunctionComponent<ICircularButtonProps> = ({
+  direction,
+  size,
+  className,
+  handlerOnClick,
+}) => {
   return (
-    <button className="circular-button">
+    <button
+      className={`circular-button ${className}`}

```

Comment:
@TaniaZaslon [This](https://github.com/AnastasiSo/Portfolio-Project/pull/16#discussion_r1113986490) Done ✔